Program.cs

```diff
@@ -1,7 +1,30 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         string[] abecedarianArr = { "glory", "floor", "ant", "victory", "aegilops", "access" };
+         int index = 0;
+         foreach (string word in abecedarianArr) {
+            if (Abecedarian (word, index))

```

Comment:
Instead of passing the index as parameter, could you check for the maxIndex within this loop? Because index is used only for assignment in the callee method.

Reply:
Changes are done.